### PR TITLE
Feature/localfavorit

### PR DESCRIPTION
**Pull request by t0bbl**
add localstorage favorit function and a small favorit site

### CODE REVIEW

File: pages/Favorit/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <>
+      <Header />
+      <Container>
+        {favPictures.map((pic) => (
+          <>
+            <PreviewPicture
+              key={pic.picSRCSlug}
+              imageSrc={pic.picSRC}
+              imageName={pic.picSRCSlug}
+            />
```
Comment (CJAkkuec) [inline pages/Favorit/index.js:28]: You may want to consider passing an `alt` value as a prop as well. 🤔 I've checked, and it appears that the `alt` property stays the same always.

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Looking good so far! 👍 Just a few small notes.

File: components/Container.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width: 120%;
```
[the remark this answers is not in the document]
Comment (t0bbl) [reply components/Container.js:4]: that might be but it works for now in the desired size. its something iam going to wrestle with when i introduce a actually responsive design.

File: components/Container.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  height: 120%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 10vw;
+  gap: 20vw;
```
[the remark this answers is not in the document]
Comment (t0bbl) [reply components/Container.js:11]: good catch, thank you!

**Review by CJAkkuec — APPROVED** [review]

**Pull request merged**